util/chpltags

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env python3
-import os, sys, re, subprocess, optparse
-from distutils.spawn import find_executable
+
+import optparse
+import os
+import subprocess
+import sys
+
+from utils import which
```

Comment:
I don't think the chplenv utils is available to this script. Given that it's a dev script requiring python3, I think you can just bring in from `shutil`